README.md

```diff
@@ -69,6 +69,8 @@ several.  In general, the test executable for
 `{Cabal,cabal-install}:$TESTNAME` will be stored at
 `dist-newstyle/build/{Cabal,cabal-install}-$VERSION/build/$TESTNAME/$TESTNAME`.
 
+To run a single test, use `-p` which applies a regex filter to the test names.
```

Comment:
`README.md` talks about `cabal new-build` only, so I'd postpone adding `--test-option`, until we talk have `new-test`.